build/preset.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Generate a Babel preset that outputs ES modules when the env is `es`, and
+ * CommonJS modules otherwise.
+ */
+
+const es = (process.env.BABEL_ENV || process.env.NODE_ENV) === 'es';
+
+module.exports = {
+  presets: [
+    ['es2015', {
+      modules: es ? false : 'commonjs',
```

Comment:
Actually why is there no way to tell babel to use a different config file?
I'd like if we could just write `babel --conf es.babelrc [...]` instead having to do this weird trick with NODE_ENV.
I looked around but can't find anything that would allow us to make this less tricky 😞